src/lib/png.ts

```diff
@@ -35,26 +35,45 @@ export const filterImagesBySizes = (images: ImageInfo[], sizes: number[]) => {
  * @param size The size (width/height) of the image.
  * @param dir Path of the file output directory.
  * @param logger Logger.
+ * @param autoMargin If true, than automargin the image.
  * @return Image generation task.
  */
 const generate = async (
   svg: Buffer,
   size: number,
   dir: string,
-  logger: Logger
+  logger: Logger,
+  autoMargin = false,
 ): Promise<ImageInfo> => {
   const dest = path.join(dir, size + '.png')
   logger.log('  Create: ' + dest)
 
+  const background = { r: 0, g: 0, b: 0, alpha: 0 };
```

Comment:
This is always transparent, so there seems no need to make it a variable.

Since this value is constant-like, it may be specified directly or as a constant in the module scope under the name "TRANSPARENT" instead of as an Object Literal.

Reply:
Yes, it will be, but it is used only here.